tests/test_more_ops.py

```diff
@@ -0,0 +1,54 @@
+import torch
+import torch_ttnn
+import ttnn
+import unittest
+
+
+class SoftmaxModule(torch.nn.Module):
```

Comment:
https://github.com/tenstorrent/pytorch2.0_ttnn/blob/main/tests/lowering/normalization/test_softmax.py

Reply:
I just removed this file as I see this as a duplicate test for softmax.